Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Lever/lever.yml

```
@@ -0,0 +1,59 @@
+    sprite: _Goobstation/Objects/Weapons/Guns/Lever/chester.rsi
+  - type: ItemSlots
+    slots:
+      gun_magazine:
+        name: Magazine
+        startingItem: MagazineMagnumLeverRifle
+        insertSound: /Audio/Weapons/Guns/MagIn/rifle_load.ogg
+        ejectSound: /Audio/Weapons/Guns/MagOut/sfrifle_magout.ogg
+        priority: 2
+        whitelist:
+          tags:
+            - MagazineMagnumLeverRifle
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)